eng/pipelines/coreclr/ci.yml

```diff
@@ -31,6 +31,52 @@ jobs:
 #
 - template: /eng/pipelines/common/checkout-job.yml
 
+#
+# Debug Corelib builds
+#
+- template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
Now that I look at this job, we should just need to build corelib for checked as libraries build only use the checked runtime. 

https://github.com/dotnet/runtime/pull/34166/files#diff-b7c9a0014b41923da8701c3dbade7868R136

Reply:
yup fixed.